### PR TITLE
new esx:pauseMenuActive handler

### PR DESCRIPTION
**Pull request by Rav3n95**

### CODE REVIEW

**Review by CsokiHUN — APPROVED** [review]

File: [core]/es_extended/client/modules/actions.lua [hunk 1]
```diff
@@ -83,20 +95,20 @@ if Config.EnableDebug then
...
 
-    AddEventHandler('esx:playerEnteringVehicleAborted', function()
-        print('esx:playerEnteringVehicleAborted')
+    AddEventHandler('esx:EnteringVehicleAborted', function()
+        print('esx:EnteringVehicleAborted')
     end)
 
-    AddEventHandler('esx:playerEnteredVehicle', function(vehicle, seat, displayName, netId)
-        print('esx:playerEnteredVehicle', 'vehicle', vehicle, 'seat', seat, 'displayName', displayName, 'netId', netId)
+    AddEventHandler('esxEnteredVehicle', function(vehicle, plate, seat, displayName, netId)
```
Comment (CsokiHUN) [inline [core]/es_extended/client/modules/actions.lua:106]: bad event name!

Comment (CsokiHUN) [reply [core]/es_extended/client/modules/actions.lua:106]: missing :

Comment (Rav3n95) [reply [core]/es_extended/client/modules/actions.lua]: fixed

File: [core]/es_extended/client/modules/actions.lua [hunk 2]
```diff
@@ -83,20 +95,20 @@ if Config.EnableDebug then
         print('esx:playerJumping')
     end)
 
-    AddEventHandler('esx:playerEnteringVehicle', function(vehicle, seat, netId)
-        print('esx:playerEnteringVehicle', 'vehicle', vehicle, 'seat', seat, 'netId', netId)
+    AddEventHandler('esx:EnteringVehicle', function(vehicle, plate, seat, netId)
```
Comment (Gellipapa) [inline [core]/es_extended/client/modules/actions.lua:98]: I think you should lowerCamelCase name convention.
for example: esx:enteringVehicle

Comment (Rav3n95) [reply [core]/es_extended/client/modules/actions.lua:98]: Sure

**Pull request merged**